crates/relayer-types/src/core/ics04_channel/version.rs

```diff
@@ -52,6 +52,10 @@ impl Version {
             })
             .unwrap_or(false)
     }
+
+    pub fn is_ics20_v2(&self) -> bool {
+        self.0.contains("ics20-2")
```

Comment:
```suggestion
        self.0 == "ics20-2"
```

Reply:
This might fail to catch an ICS29 channel with ICS20 v2 since the format would be:

```json
{
  "fee_version": "ics29-1",
  "app_version": "ics20-2"
}
```

Reply:
Ah right it can contain json, then we should probably try to deserialize the version as a JSON `Value` and look at the `app_version` field, and otherwise fall back to comparing by equality.

Reply:
Used JSON deserialization to verify the ics20 versions in this commit https://github.com/informalsystems/hermes/pull/4008/commits/f68c5e8cadf0d9c4f884b19006ce2d167e95565e